### DISCUSSION TITLE
UI/UX improvement - Adding additional details when deleting and checking

### DISCUSSION THREAD

**Discussion by jomarcenter-mjm** [ThioJoe/YT-Spammer-Purge#43] [Ideas]
Suggestion to have a way to show what comment is currently being deleted than just showing please wait. this would help out in users in knowing if the software is still working or it stuck.

Also including on the check to change from "." to Checking Comment: Remaining 100 and it will countdown. I think it will improved the User experience a bit and make it easier to know if something is happening and/or how long it working on something.

**Comment by jomarcenter-mjm** [comment]
weirdly I am working on it but currently figuring it out on how to build Python Script.

**Comment by ThioJoe** [comment]
The thing is when it deletes the comments it does them 50 at a time, so most of the waiting is just waiting for the API to respond. Still might be possible to add progress in blocks of 50. The counter for the checking should be possible though.
